<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionDialog</class>
 <widget class="QDialog" name="QuestionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Question</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="questionLabel">
     <property name="text">
      <string>This is where the question will go.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="answersContainer" native="true">
     <layout class="QVBoxLayout" name="answersLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
